ansible_image_validation/validation-playbooks/ade_packages_validation.yaml

```diff
@@ -0,0 +1,29 @@
+###
+# This Playbook checks if the packages required by Azure Disk Encryption is installed 
+###
+
+---
+
+- name: Check if the packages required by Azure Disk Encryption is installed
+  command: rpm -q "{{ item }}"
+  loop:
+    - lsscsi
+    - psmisc
+    - lvm2
+    - uuid
+    - at
+    - patch
+    - cryptsetup
+    - cryptsetup-reencrypt
+    - procps-ng
+    - util-linux
+  register: package_check
+  ignore_errors: yes
```

Comment:
I feel ignoring errors may not be a good idea

Reply:
Here, this is required as we want to perform the next set of validations also. Otherwise, the ansible will stop the playbook execution.